### PR TITLE
Create corvinus-harvard.csl

### PR DESCRIPTION
**Pull request by VERESS60**

### CODE REVIEW

**Comment by rmzelle** [comment]
Is this for the Corvinus University of Budapest? Is there an online style guide for your university (or department) that describe this citation format?

**Comment by VERESS60** [comment]
Yes, the purpose is to provide students with a Zotero version which is line with the citation requirements of te University. 
Actually can you advise how can we change after the Title the "," to "."?
Br
József
 

    On Sunday, April 14, 2019, 9:17:48 PM GMT+2, Rintze M. Zelle <notifications@github.com> wrote:  
 
 
Is this for the Corvinus University of Budapest? Is there an online style guide for your university (or department) that describe this citation format?

File: corvinus-harvard.csl [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    <sort>
+      <key macro="author"/>
+      <key macro="issued-year" sort="ascending"/>
+    </sort>
+    <layout suffix=".">
+      <group>
+        <text macro="author" suffix="."/>
+        <text macro="issued" suffix=" "/>
+        <group delimiter=", ">
+          <text macro="title"/>
```
Comment (rmzelle) [inline corvinus-harvard.csl]: > Actually can you advise how can we change after the Title the "," to "."?

It originates from the `<group delimiter=", ">` on line 302 (although this delimiter is also used for separating the locators from whatever precedes it, so you can't just change this value by itself.

**Comment by rmzelle** [comment]
> citation requirements of t[h]e University. 

Are those available online? Or only in print? We try to include a "documentation" link for all our styles (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link)

And s this style for the entire university of a single department?

**Comment by VERESS60** [comment]
For the time being it was used as a file that the users could copy .Our understanding was that this variant after tested could be added to the Zotero similar to (many) other university styles available to all users.
The proposal is to use this variant at least in the business school of the university - but hopefully also in the two other faculties.

    On Thursday, April 18, 2019, 6:01:12 AM GMT+2, Rintze M. Zelle <notifications@github.com> wrote:  
 
 

citation requirements of t[h]e University.


Are those available online? Or only in print? We try to include a "documentation" link for all our styles (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link)

And s this style for the entire university of a single department?

**Comment by rmzelle** [comment]
> For the time being it was used as a file that the users could copy .Our understanding was that this variant after tested could be added to the Zotero similar to (many) other university styles available to all users.

I think you're slightly misunderstanding my question.

Yes, we should be able to accept this style, but I would first like to know more about where the citation requirements are written down. Is there a university webpage that describes the citation format? Or is there a printed style guide?

**Comment by VERESS60** [comment]
The requirements are written in the study and examination rules of the Corvinus University and are available on its website:
https://www.uni-corvinus.hu/fileadmin/user_upload/en/faculty_of_social_sciences/III_1_TVSZ_2016_junius_20_final_english_clean-1.pdf
I also send you attached the English version. It explains rules of citation and  provides examples (for example on pages 94-95).
    On Tuesday, April 23, 2019, 4:40:41 AM GMT+2, Rintze M. Zelle <notifications@github.com> wrote:  
 
 

For the time being it was used as a file that the users could copy .Our understanding was that this variant after tested could be added to the Zotero similar to (many) other university styles available to all users.


I think you're slightly misunderstanding my question.

Yes, we should be able to accept this style, but I would first like to know more about where the citation requirements are written down. Is there a university webpage that describes the citation format? Or is there a printed style guide?

File: corvinus-harvard.csl [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper" match="any">
+        <group delimiter=", ">
+          <group>
+            <text variable="volume" prefix="Vol. "/>
+            <text variable="issue" prefix=" No. "/>
+          </group>
+          <group>
+            <label plural="always" variable="locator" form="symbol"/>
```
Comment (adam3smith) [inline corvinus-harvard.csl]: I think you'll want something like this:

```suggestion
            <label plural="always" variable="page" form="short" suffix=" "/>
```

**Comment by adam3smith** [comment]
Apart from my one suggestion this looks good to me code-wise. @rmzelle leaving metadata to you.

**Comment by rmzelle** [comment]
> The requirements are written in the study and examination rules of the Corvinus University and are available on its website: https://www.uni-corvinus.hu/fileadmin/user_upload/en/faculty_of_social_sciences/III_1_TVSZ_2016_junius_20_final_english_clean-1.pdf

Thanks. That's exactly what we were looking for.

The style is currently locked to English, and since Corvinus is a Hungarian University, we would typically add "(English)" to the style title to make this clear to the user. Would "Corvinus Business School (English)" be a good name for this style? Or should it be something like "Corvinus University of Budapest - Faculty of Applied Economics (English)" or just "Corvinus University of Budapest (English)"?

**Comment by adam3smith** [comment]
@VERESS60 once you answer @rmzelle 's final questions, we would be able to add this to the repository:

>The style is currently locked to English, and since Corvinus is a Hungarian University, we would typically add "(English)" to the style title to make this clear to the user. Would "Corvinus Business School (English)" be a good name for this style? Or should it be something like "Corvinus University of Budapest - Faculty of Applied Economics (English)" or just "Corvinus University of Budapest (English)"?

**Comment by VERESS60** [comment]
The name of the proposed Zotero variant could be: Corvinus-Harvard English /Hungarian
It hopefully will be used by both students and professors for "handling" English as well as Hungarian sources and literature.
Best regards:
József Veress
    On Friday, June 7, 2019, 02:58:53 AM GMT+2, Sebastian Karcher <notifications@github.com> wrote:  
 
 
@VERESS60 once you answer @rmzelle 's final questions, we would be able to add this to the repository:


The style is currently locked to English, and since Corvinus is a Hungarian University, we would typically add "(English)" to the style title to make this clear to the user. Would "Corvinus Business School (English)" be a good name for this style? Or should it be something like "Corvinus University of Budapest - Faculty of Applied Economics (English)" or just "Corvinus University of Budapest (English)"?

**Pull request closed without merging**